student-info-system/sis/migrations/0002_academicprobationtask_interval_tasks.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 3.1.7 on 2021-04-15 20:29
```

Comment:
Could you please regen the db to remove the migrations?

Reply:
done